src/main/java/com/synopsys/integration/jenkins/scan/service/ScanCommandsFactory.java

```diff
@@ -54,26 +124,47 @@ public static Map<String, Object> preparePipelineParametersMap(SecurityScanStep
         if (!Utility.isStringNullOrBlank(scanStep.getBlackduck_scan_failure_severities())) {
             parametersMap.put(ApplicationConstants.BLACKDUCK_SCAN_FAILURE_SEVERITIES_KEY, scanStep.getBlackduck_scan_failure_severities().toUpperCase());
         }
-        if (!Utility.isStringNullOrBlank(scanStep.getBitbucket_token())) {
-            parametersMap.put(ApplicationConstants.BITBUCKET_TOKEN_KEY, scanStep.getBitbucket_token());
-        }
-
         parametersMap.put(ApplicationConstants.BLACKDUCK_SCAN_FULL_KEY, scanStep.getBlackduck_scan_full());
         parametersMap.put(ApplicationConstants.BLACKDUCK_AUTOMATION_FIXPR_KEY, scanStep.getBlackduck_automation_fixpr());
         parametersMap.put(ApplicationConstants.BLACKDUCK_AUTOMATION_PRCOMMENT_KEY, scanStep.getBlackduck_automation_prcomment());
+    }
+
+    private static void prepareCoverityParametersMap(SecurityScanStep scanStep, Map<String, Object> parametersMap) {
```

Comment:
Better approach might be returning a new map, rather then mutating the method parameter.

Reply:
Updated accordingly